frontend/storage/utils/apiRequestManager.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import { SERVER_ADRESS } from '../common';
+
+export async function apiRequestGet(endpoint: string, queryParams: { [key: string]: any }) {
```

Comment:
The following might seem unnecessary at the moment but we know we'll need at least a function to POST data to the server and it'll help keeping things clean.

Make this function either a part of a class or a constant object called something like `ApiRequests`.